src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -981,6 +982,7 @@ public void AllWellKnownTypeMembers()
                     case WellKnownMember.System_MemoryExtensions__SequenceEqual_ReadOnlySpan_T:
                     case WellKnownMember.System_MemoryExtensions__AsSpan_String:
                     case WellKnownMember.System_Runtime_CompilerServices_CompilerFeatureRequiredAttribute__ctor:
+                    case WellKnownMember.System_Diagnostics_CodeAnalysis_UnscopedRefAttribute__ctor:
```

Comment:
I assume this is a temporary thing?

Reply:
I'm asking as it relates to the comment below it - `// Not yet in the platform.`

Reply:
The type is not in the set of assemblies we're using for this test. I suspect the comment below is meant to indicate that.